internal/scan-manager/core/src/main/java/org/wso2/security/tools/scanmanager/core/dao/UserDAO.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright (c) 2019, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.wso2.security.tools.scanmanager.core.dao;
+
+import org.springframework.data.repository.PagingAndSortingRepository;
+import org.springframework.stereotype.Repository;
+import org.wso2.security.tools.scanmanager.common.external.model.User;
+
+/**
+ * The DAO class that manage the persistence methods of the users.
+ */
+@Repository
+public interface UserDAO extends PagingAndSortingRepository<User, Integer> {
+
+    /**
+     * Get User by user id.
+     *
+     * @param id user id assigned for a user
+     * @return user object for the given user id
+     */
+    public User getById(String id);
```

Comment:
Why isn't this exposed through the UserService? Would other code directly call this method without going through the service?